docs/sample_config.yaml

```
@@ -978,6 +978,8 @@ metrics_flags:
 # Whether or not to report anonymized homeserver usage statistics.
 # report_stats: true|false
 
+# The endpoint where to report the anonymized homeserver usage statistics to. Defaults to matrix.org
+# report_stats_endpoint: "https://matrix.org/report-usage-stats/push"
```

code_style.rst has some words on the correct formatting here. (Note that a number of the existing settings do not follow the correct format, so don't blindly copy them)

```suggestion
#
#report_stats_endpoint: https://example.com/report-usage-stats/push
```